src/Domain/CoordinateMaps/Translation.hpp

```diff
@@ -22,6 +22,11 @@ class Translation {
  public:
   static constexpr size_t dim = 1;
 
+  // required for clang-xcode7.3 error:
+  // default initialization of an object of const type without a
+  // user-provided default constructor
+  Translation() = default;
```

Comment:
You don't need this.  Just add `{}` after the variable name on the line indicated in the error message.